src/main/app/claims/models/partyStatement.ts

```diff
@@ -0,0 +1,24 @@
+
+import { Serializable } from 'models/serializable'
+import { Offer } from 'claims/models/offer'
+
+export class PartyStatement implements Serializable<PartyStatement> {
+  type: string
+  madeBy: string
+  offer?: Offer
+
+  constructor (type: string, madeBy: string, offer?: Offer) {
+    this.type = type
+    this.madeBy = madeBy
+    this.offer = offer
+  }
+
+  deserialize (input: any): PartyStatement {
+    if (input) {
+      this.type = input.type
+      this.madeBy = input.madeBy
+      this.offer = new Offer().deserialize(input.offer)
```

Comment:
If offer is optional I think we should have a condition here so that when undefined was returned from backend we have `PartyStatement.undefined `. I think about something like:

```
if (input.offer) {
  this.offer = new Offer().deserialize(input.offer)
}
```